test/helper.rb

```diff
@@ -15,36 +20,8 @@
   end
 end
 
-ENV['RACK_ENV'] = ENV['RAILS_ENV'] = 'test'
-
-trap 'TSTP' do
```

Comment:
I haven't found any reason to trap `TSTP` signal in tests

Reply:
It's useful for debugging deadlocked tests, I had to use it years ago so it's ok to remove.